src/main/java/de/photon/anticheataddition/events/ModulePlayerEvent.java

```diff
@@ -5,13 +5,20 @@
 import org.bukkit.entity.Player;
 import org.bukkit.event.Cancellable;
 import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
 
 @Getter
 public abstract class ModulePlayerEvent extends Event implements Cancellable
 {
     protected final Player player;
     protected final String moduleId;
     private boolean cancelled = false;
+    protected static final HandlerList handlers = new HandlerList();
```

Comment:
This will cause all subclasses to have the exact same HandlerList as it is static, which will decrease performance severely, as a lot of event listeners will be checked that are not of the same class.